src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -692,10 +720,31 @@ private boolean isEmpty(List<?> l) {
         }
 
         @Override
-        public List<CreateTableRequest> getPhysicalTables() {
-            return createTableRequests;
+        public List<CreateTableRequest> getStaticPhysicalTables() {
+            return staticPhysicalTables;
+        }
+
+        @Override
+        public CreateTableRequest getDynamicPhysicalTable(DynamoTableDescription virtualTable) {
```

Comment:
This is called when we want to create a new multitenant table, which means we create a new physical table dynamically. To do this, we convert each table or secondary index PK of the virtual table into a physical PK, according to how the TablePartitioningStrategy defines the conversion.